packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,21 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const noPublishDateContentTypes = ['event', 'webinar', 'contact'];
 
 <endeavor-breadcrumbs-website-section section=content.primarySection />
 <cms-content-name tag="h1" class="page-wrapper__title" obj=input.content />
 <cms-content-teaser tag="p" class="page-wrapper__deck" obj=input.content />
+<if(content.type === 'event')>
+    <endeavor-content-starts obj=input.content />
+    <endeavor-content-ends class="page-wrapper__ends-date" obj=input.content/>
```

Comment:
One last, minor change...

If you remove `class="page-wrapper__ends-date"` and instead add `block="item"` to both `<endeavor-content-starts>` (L10 and L14) and `<endeavor-content-ends>` (L11) you can remove the `&__ends-date` styles from the `themes/pennwell/styles/components/_page.scss` file.

I realized that this styling is already handled by the item block classes, so we might as well reuse them.

Reply:
pushed